src/content/docs/dropins/cart/index.mdx

```diff
@@ -60,4 +60,4 @@ The Cart dropin does not provide any slots.
 
 ### Functions
 
-Describes the API functions available in the Cart dropin: `addProductsToCart`, `createEmptyCart`, `fetch-graphql`, `getCartData`, `getEstimateShipping`, `getEstimatedTotals`, `getStoreConfig`, `graphql`, `initialize`, `initializeCart`, `resetCart` and `updateProductsFromCart`. These functions allow developers to retrieve and display detailed cart information dynamically. Visit the [Cart Functions](/dropins/cart/cart-functions/) page to learn more.
+Describes the API functions available in the Cart dropin: `addProductsToCart`, `applyCouponsToCart`, `createEmptyCart`, `fetch-graphql`, `getCartData`, `getEstimateShipping`, `getEstimatedTotals`, `getStoreConfig`, `graphql`, `initialize`, `initializeCart`, `resetCart` and `updateProductsFromCart`. These functions allow developers to retrieve and display detailed cart information dynamically. Visit the [Cart Functions](/dropins/cart/cart-functions/) page to learn more.
```

Comment:
Not a blocker, but we need to rethink how we list functions in this section. This isn't easy to read on the page. After the number of functions reaches a certain number, I think I'd prefer to change this to some generic text or general text applicable to the Cart experience.

Reply:
Deleted the list entirely. There's no reason to manually maintain such a list on an index page